charts/tensorleap/charts/node-server/templates/deployment.yaml

```diff
@@ -154,5 +157,8 @@ spec:
                   name: minio-secret
                   key: rootPassword
       volumes:
+        - name: minio
+          persistentVolumeClaim:
+            claimName: tensorleap-minio
```

Comment:
so now both minio service and node write to this volume? didn't it should be changed to ReadWriteMany?

Reply:
No need to change it when they are both running on the same node